stories/index.ts

```diff
@@ -9,5 +9,6 @@ require('./LabelWithOptions.story');
 require('./Text').story();
 require('./ToggleSwitch').story();
 require('./Tooltip').story();
+require('./GlobalHelper');
```

Comment:
You don't like my `StorySettings`?
I'd like to have it as a pattern, so we can import the StorySettings from the e2e and: build the storyUrl, later maybe build url for specific examples, and we can have also some info about the examples so that we can test them.
Having StorySettings in the story itself is sometimes a problem, because if we import the story file from the e2e, then there could be runtime errors (e2e runs in Node, and story runs in browser), so it is good to keep the story setting in a separate file. Thus, a story folder is a proper solution.

What is the benefit of the flatten convension?